### PR TITLE
Rename ambigious collision object filter in planning scene

### PR DESCRIPTION
**Pull request by davetcoleman**
While debugging the planning scene, i find it very hard to understand what everything is doing. Simple cleanup.

### CODE REVIEW

**Comment by IanTheEngineer** [comment]
What if we split the difference and called it `collision_object_tf_filter` or `collision_object_transform_filter` at the risk of being too wordy?

**Comment by davetcoleman** [comment]
I'm trying to cleanup and understand the planning scene (its pretty complex) but this particular change is not that important to me.
